index.html

```diff
@@ -97,7 +97,122 @@ <h2>Software Developer</h2>
 
             </ul>
         </section>
+        <section class="detailContent">
+            <h2 class="worktitile"> MY RECENT WORK</h2>
+            <ul class="works">
+               
```

Comment:
### **You are missing these images in the background :mag:**
![missing-background](https://user-images.githubusercontent.com/93039359/176911515-eca34ef5-6fdd-4f38-8cd5-8efef82986a2.png)
![missing-background-2](https://user-images.githubusercontent.com/93039359/176911541-bd4daf10-1361-4f6d-a8e2-309974a94750.png)
![missing-background-3](https://user-images.githubusercontent.com/93039359/176911554-dc925c57-1e8f-4ef9-a3c4-1c05215d012f.png)

- [ ] Please add these items in your background :+1: